### ISSUE TITLE
Using wrong throats in many models

### ISSUE THREAD

**Issue by jgostick** [PMEAL/OpenPNM#967]
You've put a major bug into many of the models!

This is the code in the models now (e.g. throat_equivalent_area.spherical_pores):

```
    network = target.project.network
    # Note, I change the mapping api below, but this is not the problem
    throats = network.map_throats(throats=target.Ts, target=target)  
    cn = network['throat.conns'][throats]
    d1 = network[pore_diameter][cn[:, 0]]
    d2 = network[pore_diameter][cn[:, 1]]
    # This problem is here.  You use throats from the network but are retrieving throat_conduit_lengths
    # from the target, which is not the same length as the network where there are 2 geometries
    L1 = target[throat_conduit_lengths+'.pore1'][throats]
    L2 = target[throat_conduit_lengths+'.pore2'][throats]
```

The strange part about this that this error is only showing up in ONE docstring!  How do we not have unit tests to check for the impact of multiple subdomains on all models?!

**Comment by ma-sadeghi** [comment]
This is addressed in PR #968. Will make tests in the morning :)

**Comment by jgostick** [comment]
fixed in PR #968
